extensions/standard-processors/tests/unit/ListenSyslogTests.cpp

```diff
@@ -439,12 +439,13 @@ TEST_CASE("Test ListenSyslog via TCP with SSL connection", "[ListenSyslog]") {
 
   SingleProcessorTestController controller{listen_syslog};
   auto ssl_context_service = controller.plan->addController("SSLContextService", "SSLContextService");
+  const auto executable_dir = minifi::utils::file::FileUtils::get_executable_dir();
   REQUIRE(controller.plan->setProperty(ssl_context_service, controllers::SSLContextService::CACertificate.getName(),
-    minifi::utils::file::FileUtils::get_executable_dir() + "/resources/ca_cert.crt"));
+    minifi::utils::file::concat_path(executable_dir, "resources/ca_cert.crt")));
```

Comment:
We need `concat_path` between `resources` and `ca_cert.crt`, too.  Maybe it's time for a 3-parameter (or variadic) `concat_path`?

Reply:
I checked on Windows and it works with the current solution, it can evaluate the path this way as well, it is similarly done in the SFTP tests. It is a good idea to change the concat_path to have variadic arguments, but I think it should be done outside the scope of this PR. I created a separate Jira ticket for this task https://issues.apache.org/jira/browse/MINIFICPP-1911